cmd/conduit/main.go

```diff
@@ -80,6 +83,8 @@ func parseConfig() conduit.Config {
 	cfg.DB.Postgres.Table = stringPtrToVal(dbPostgresTable)
 	cfg.GRPC.Address = stringPtrToVal(grpcAddress)
 	cfg.HTTP.Address = stringPtrToVal(httpAddress)
+	cfg.Log.Level = stringPtrToVal(level)
+	cfg.Log.Format = stringPtrToVal(format)
```

Comment:
we would still worry about the case only in the tests, so we have to make sure to use "json" not "JSON" and so on

Reply:
@maha-hajja I believe you are talking about the config tests right? Making this case insenstive is not a must-have IMHO, but if we want that, than it looks like that means parsing needs to be pulled into the config struct.

Reply:
I would like to make it insensitive, because people tend to use uppercase for "JSON" and log levels ("INFO"), so it would just remove some friction there.

> we would still worry about the case only in the tests

We don't have to worry about that, we can say that the `Config` expects the strings to be lower case, otherwise it's invalid. And we could ensure case insensitivity by converting the CLI flags to lower case before setting them in the `Config`, that's all I'm saying.